tests/utils/profilehandler/interface.go

```diff
@@ -0,0 +1,73 @@
+package profilehandler
+
+type Profile struct {
+	Name              string             `yaml:"as,omitempty"`
+	Version           string             `yaml:"version,omitempty"`
+	ChannelGroup      string             `yaml:"channel_group,omitempty"`
+	Region            string             `yaml:"region,omitempty"`
+	NamePrefix        string             `yaml:"name_prefix,omitempty"`
+	ClusterConfig     *ClusterConfig     `yaml:"cluster,omitempty"`
+	AccountRoleConfig *AccountRoleConfig `yaml:"account-role,omitempty"`
+}
+type AccountRoleConfig struct {
+	Path               string `yaml:"path,omitempty"`
+	PermissionBoundary string `yaml:"permission_boundary,omitempty"`
+}
+type ClusterConfig struct {
```

Comment:
Leaving a nore for myself, we have a 'Spec' struct for creating cluster, probably be worth looking at combining both of these structs at some point and move to the ocm-common

Reply:
Yes. I believe so. Once it is moved to ocm-common. Then we can reuse it in backend testing and also rhcs.